data/json/items/armor/boots.json

```diff
@@ -145,6 +145,48 @@
     "environmental_protection": 2,
     "flags": [ "VARSIZE", "WATERPROOF", "STURDY" ]
   },
+  {
+    "id": "foot_protectors_eod",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "pair of EOD foot protectors", "str_pl": "pairs of EOD foot protectors" },
+    "description": "Armored foot protectors constructed from steel and nomex for explosive ordnance disposal.  They are designed to protect against overpressure, fragmentation, impact and heat.",
+    "weight": "1220 g",
+    "volume": "3 L",
+    "price": 50000,
+    "covers": [ "FEET" ],
+    "coverage": 100,
+    "encumbrance": 40,
+    "warmth": 10,
+    "material": [ "steel", "nomex" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "environmental_protection": 2,
+    "material_thickness": 7,
+    "flags": [ "STURDY", "OUTER" ]
+  },
+  {
+    "id": "toecaps",
+    "type": "ARMOR",
+    "name": "pair of toecaps",
+    "name_plural": "pairs of toecaps",
```

Comment:
There are still some occurrences of `name_plural` strings instead of `name` object with `str_pl` inside.